### PR TITLE
Fix documentation bug 

### PR DESCRIPTION
**Pull request by mushroot**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

- About the projects https://github.com/orgs/laminas/projects/8#card-36888172
this is my way to set value options from database for a select element

### CODE REVIEW

**Comment by froschdesign** [comment]
@mushroot 
> this is my way to set value options from database for a select element

1. The description for application integration must contain only basic example. Everything else should be described separated. The plan is to create a "Cookbook" section like in [Mezzio](https://docs.mezzio.dev/mezzio/v3/cookbook/autowiring-routes-and-pipelines/).
2. In the controller is not the best idea because it means to duplicate code if the form is used multiple times. A [custom element](https://discourse.laminas.dev/t/handling-laminas-forms-with-dependencies-the-view-results/1343/12?u=froschdesign) can be used or the `Laminas\Db\Adapter\AdapterAwareTrait` and a delegator.

File: docs/book/application-integration/usage-in-a-laminas-mvc-application.md [hunk 1]
```diff
@@ -32,11 +32,20 @@ class AlbumForm extends Form implements InputFilterProviderInterface
         // Title
         $this->add([
             'name'    => 'title',
-            'type'    => Text:class,
+            'type'    => Text::class,
```
Comment (froschdesign) [inline docs/book/application-integration/usage-in-a-laminas-mvc-application.md:35]: Good catch! 👍

**Review by froschdesign — CHANGES_REQUESTED** [review]
The bugfix is good but everything else should be handled separately and therefore reverted. Thanks in advance!

File: docs/book/application-integration/usage-in-a-laminas-mvc-application.md (hunk 1 above)
Comment (mushroot) [reply docs/book/application-integration/usage-in-a-laminas-mvc-application.md:35]: ok, I reverted it.

**Review by froschdesign — APPROVED** [review]

**Comment by Slamdunk** [comment]
Thank you

**Pull request merged**